src/components/Testimonials/Testimonials.css

```diff
@@ -0,0 +1,48 @@
+.testimonials {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 90px 60px 50px;
+  gap: 10px;
+  color: #ffffff;
+}
+
+.testimonialsHead {
+  padding-bottom: 40px;
+}
+
+.testimonialsTitle {
+  text-align: center;
+  font-size: 52px;
+}
+
+.testimonialsItems {
+  display: flex;
+  flex-direction: row;
+}
+
+.testimonialCardDiv {
+  display: table;
+  align-items: center;
+  padding: 10px;
```

Comment:
using table display here is not exactly ideal.
Think about the responsiveness aspect of your site.

you are yet to implement responsiveness, this might make it difficult later on.